build/common.sh

```diff
@@ -93,7 +93,7 @@ readonly KUBE_CONTAINER_RSYNC_PORT=8730
 #
 # $1 - server architecture
 kube::build::get_docker_wrapped_binaries() {
-  local debian_iptables_version=v12.1.2
```

Comment:
no idea... is this something networking folks would have input on?

Reply:
I believe the request originally came from Network?
https://github.com/kubernetes/kubernetes/issues/94700

Reply:
Yep, the arbitrary versions were pretty confusing and have caused issues in the past.
I fixed the tags up to be more descriptive in https://github.com/kubernetes/release/pull/1496.